docs/user_docs/concepts/intent

```diff
@@ -0,0 +1,90 @@
+# Intent
+
+Intent represents the operational intentions you want to deliver with Kusion. The intentions here is expected to contain all components during the SDLC, such as resources, dependencies and policies. All AppConfigurations written by application developers will also be converted into the Intent by Kusion module generators before Kusion acutally perform operations.
```

Comment:
Intent describes what you expect your infrastructure to look like: which service to run, the load-balancer to setup, the database schema to apply, and so on. Kusion Module Generators take care of taking any AppConfigurations and massage it into Intent. Once Intent is generated,  Kusion Engine takes care of updating production state to match the Intent.